test/bundles/bundles.test.ts

```diff
@@ -53,6 +53,7 @@ class BpmnStaticPageSvgTester extends BpmnPageSvgTester {
     super({ ...targetedPageConfiguration, diagramSubfolder: 'none' }, page);
   }
   override async gotoPageAndLoadBpmnDiagram(): Promise<void> {
+    // eslint-disable-next-line -- We don't want to change a working configuration
```

Comment:
**question:** there is no rule specified here, do we really want to disable all estlint rules? This seems too much